spring-cloud-gcp-docs/src/main/asciidoc/spanner.adoc

```diff
@@ -885,6 +885,20 @@ point in time of the transaction.
 Because read-only transactions are non-locking and can be performed on points in time in the past, these
 are recommended for functions that do not perform write operations.
 
+===== @Transactional and declerative transaction management
```

Comment:
Spelling: "declarative".

I would title it
**Declarative Transaction Management and `@Transactional`**